t/feature_encrypt/07-open_unencrypted_db_with_key.t

```diff
@@ -0,0 +1,54 @@
+# vim:set ft= ts=4 sw=4 et:
+
+use Test::Nginx::Socket::Lua;
+use Cwd qw(cwd);
+
+repeat_each(2);
+
+plan tests => repeat_each() * blocks() * 5;
+
+my $pwd = cwd();
+
+our $MainConfig = qq{
+    lmdb_environment_path /tmp/test.mdb;
```

Comment:
don't rely on prior test leftovers in other test, instead, generate such mdb directory and check it in git. use the checked-in fixtures to test this case. same applies to test `06` and `08`.

Reply:
fix